File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.js
function solution(num_list) {
    var odd = 0;
    var even = 0;
    for(let i=0; i<num_list.length; i+=2){
        even+=num_list[i]
    }
    for(let i=1; i<num_list.length; i+=2){
        odd+=num_list[i]
    }
    return even>odd?even:odd
}

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.java
import java.util.Arrays;
import java.util.stream.IntStream;
class Solution {
    public int solution(int[] num_list) {
        int v1 = IntStream.range(0,num_list.length).filter(i->i%2==0).map(i->num_list[i]).sum();
        int v2 = IntStream.range(0,num_list.length).filter(i->i%2==1).map(i->num_list[i]).sum();
        
        return (v1>v2)?v1:(v2>v1)?v2:v1;
    }
}